diff --git a/src/logger/utils/streams/DetoxJSONLParser.js b/src/logger/utils/streams/DetoxJSONLParser.js
index d94743237d1e1168866d3e5aaed37f01d4398ebb..d7f44f4bd7c0616a8b6b68cc19a4c09c9dc04e05 100644
--- a/src/logger/utils/streams/DetoxJSONLParser.js
+++ b/src/logger/utils/streams/DetoxJSONLParser.js
@@ -1,4 +1,5 @@
-const JsonlParser = require('stream-json/jsonl/Parser');
+const JsonlParserModule = require('stream-json/jsonl/parser.js');
+const JsonlParser = JsonlParserModule.default ?? JsonlParserModule;
 
 const { through } = require('./transformers');
 
@@ -10,7 +11,7 @@ class DetoxJSONLParser {
   createTransformer() {
     const log = this._log;
     const readable = through();
-    const writable = JsonlParser.make({ checkErrors: true })
+    const writable = JsonlParser.asStream({ checkErrors: true })
       .on('error', (err) => {
         /* istanbul ignore else */
         if (err instanceof SyntaxError) {
